### PR TITLE
feat: add V1 volume and deposits count to /deposits/stats (stage)

### PR DESCRIPTION
**Pull request by amateima**

### CODE REVIEW

File: src/modules/deposit/service.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { Cache } from "cache-manager";
 import { Deposit } from "../scraper/model/deposit.entity";
 import { getAvgFillTimeQuery, getTotalDepositsQuery, getTotalVolumeQuery } from "./adapter/db/queries";
 
-export const GENERAL_STATS_CACHE_KEY = "stats:general";
+export const DEPOSITS_STATS_CACHE_KEY = "deposits:stats";
```
Comment (amateima) [inline src/modules/deposit/service.ts:9]: Found a better naming for the cache key

**Review by nicholaspai — APPROVED** [review]
I assume these are numbers that Chase sent you so LGTM

**Comment by nicholaspai** [comment]
Also @amateima it looks like there are some linting options
Screen Shot 2022-11-21 at 06 03 51

**Comment by amateima** [comment]
> I assume these are numbers that Chase sent you so LGTM

Yes

**Pull request merged**